### PR TITLE
Fix missing output of exec-host, fixes #2477

### LINKED ISSUE #2477: exec-host output is not shown in console
**Describe the bug**
The output of an exec-host hook is not printed to the console.

**To Reproduce**
Add to config
```
hooks:
  post-start:
  - exec: "echo hi there && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done"
  - exec-host: "echo host there && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done"
```

(Re)start ddev project

Only the output of the "exec" command is show; the "exec-host" output is not shown:

> Executing post-start hook... 
=== Running task: Exec command 'echo hi there && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done' in container/service 'web', output below 
hi there
1: Mon Aug 24 13:23:03 UTC 2020
2: Mon Aug 24 13:23:04 UTC 2020
3: Mon Aug 24 13:23:05 UTC 2020
4: Mon Aug 24 13:23:06 UTC 2020
5: Mon Aug 24 13:23:07 UTC 2020
=== Running task: Exec command 'echo host there && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done' on the host (xxx), output below 
Restarted xxxx 
Your project can be reached at xxxx

**Expected behavior**
Output for both commands should be visible in the console

**Version and configuration information (please complete the following information):**

- Host computer OS and Version: macOS Catalina
- Docker Desktop version if on macOS or Windows (from "About Docker Desktop")
- ddev version: 1.15.3
- config.yaml: see above
- Do you have any custom configuration (nginx, php, mysql) in the .ddev folder? No

**Additional context**
It's probably an side-effect from #2214 #2335 where the output was changed to stream

### PR DESCRIPTION
**Pull request by gilbertsoft**
## The Problem/Issue/Bug:
Output of exec-host is not correctly handled see issue.

## How this PR Solves The Problem:
exec.RunInteractiveCommand() is now used for exec-host tasks which solves the problem.

## Manual Testing Instructions:
See also example from the issue.
```yaml
hooks:
  post-start:
  - exec-host: "echo test exec-host && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done"
  - exec: "echo test exec && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done"
```

```bash
Executing post-start hook...
=== Running task: Exec command 'echo test exec-host && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done' on the host (NB-SIMON), output below
test exec-host
1: Sa, 24. Okt 2020 16:29:22
2: Sa, 24. Okt 2020 16:29:23
3: Sa, 24. Okt 2020 16:29:24
4: Sa, 24. Okt 2020 16:29:25
5: Sa, 24. Okt 2020 16:29:27
=== Running task: Exec command 'echo test exec && for item in 1 2 3 4 5; do echo $item: $(date); sleep 1; done' in container/service 'web', output below
test exec
1: Sat Oct 24 14:29:29 UTC 2020
2: Sat Oct 24 14:29:30 UTC 2020
3: Sat Oct 24 14:29:31 UTC 2020
4: Sat Oct 24 14:29:32 UTC 2020
5: Sat Oct 24 14:29:33 UTC 2020
```

## Related Issue Link(s):
Fixes #2477

### CODE REVIEW

File: pkg/ddevapp/task.go [hunk 1]
```diff
@@ -78,12 +80,10 @@ func (c ExecHostTask) Execute() error {
...
-	cmd := exec.Command(bashPath, "-c", c.exec)
-	var stdout bytes.Buffer
-	var stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	err = cmd.Run()
+	args := []string{"-c"}
+	args = append(args, c.exec)
+
+	err = exec.RunInteractiveCommand(bashPath, args)
```
Comment (rfay) [inline pkg/ddevapp/task.go:86]: Looks pretty elegant to me!

Comment (gilbertsoft) [reply pkg/ddevapp/task.go:86]: Or this should also work
```
args := []string{
    "-c",
    c.exec,
}
```

Comment (gilbertsoft) [reply pkg/ddevapp/task.go:86]: I did not know there is such a big difference for the two solutions, the second one safes 43 OpCodes so we should definitively go with it.
grafik

**Review by rfay — APPROVED** [review]
This is perfect, thanks so much for taking this on. It also works when interactive input is required.

**Pull request merged**